src/BlockHorizons/BlockPets/commands/ChangePetNameCommand.php

```diff
@@ -22,8 +22,14 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 			return true;
 		}
 
-		if(!$sender instanceof Player) {
+		if(!isset($args[0])){
```

Comment:
There are quite a bunch of those tiny formatting problems. I'm fine with merging it and reformatting it later but it'd be very much appreciated if you could fix those. 😜